app/src/main/java/com/example/democalculator/MainActivity.kt

```diff
@@ -1,11 +1,67 @@
 package com.example.democalculator
 
-import androidx.appcompat.app.AppCompatActivity
+import android.annotation.SuppressLint
 import android.os.Bundle
+import android.view.View
+import android.widget.Button
+import androidx.appcompat.app.AppCompatActivity
+import kotlinx.android.synthetic.main.activity_main.*
+
+class MainActivity : AppCompatActivity(), View.OnClickListener {
+    private var operand = ""
+    private var operator = ""
+    private val numbers: HashSet<String> =
+        hashSetOf("0", "1", "2", "3", "4", "5", "6", "7", "8", "9")
+    private var operators: HashSet<String> = hashSetOf("+", "-", "x", ":")
 
-class MainActivity : AppCompatActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         setContentView(R.layout.activity_main)
     }
-}
\ No newline at end of file
+
+    @SuppressLint("SetTextI18n")
+    override fun onClick(p0: View?) {
+        textTitle.visibility = View.GONE
+        val clicked: Button = p0 as Button
```

Comment:
cái này em check p0 là button trước hãng ép kiểu không lỗi 
```suggestion
        val clicked = if(p0 is Button) (p0 as Button) else return 
```